src/quill-editor.component.ts

```diff
@@ -116,7 +117,7 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
         range: range,
         oldRange: oldRange,
         source: source,
-        bounds: this.bounds || document.body
+        bounds: this.bounds ||  document.body
```

Comment:
Sorry, this change is from VScode (my linter in autosave). There is not relevant to the commit.